TrackerExtenstion.cs

```diff
@@ -38,14 +39,48 @@ public void Startup(StartupParams sp)
         /// </param>
         public void Ready(ReadyParams rp)
         {
+            // @todo Subscribe to an event after the Dynamo home screen is shown
+
+            // Display a MessageBox to the user
+            // @todo Use a view framework to improve the UI/UX
             string message = "By pressing OK you agreeing to Binoculars 🔍 data collection.";
             string title = "Terms of Use Agreement";
-
             MessageBox.Show(message, title, MessageBoxButton.OK, MessageBoxImage.Exclamation);
+
+            // @todo if user does not consent, don't store..
+            if (true) {
```

Comment:
instead of wrapping a whole bunch of code in an `if` statement, bail early.
```suggestion
            if (!true) return;
```

Reply:
Very good point! I primarily work with javascript lately so I'd completely forgotten to take advantage  of blocking/synchronous methods

Reply:
Closing in favour of issue https://github.com/teamtreedyn/Binoculars/issues/20